distribution/lib/Standard/Table/0.0.0-dev/src/Columns_To_Add.enso

```diff
@@ -0,0 +1,29 @@
+from Standard.Base import all
+from Standard.Base.Metadata import make_single_choice, Widget
+
+## PRIVATE
+type Columns_To_Add
+    ## Read all rows.
+    All_Columns
+
+    ## Read the first specified number of columns.
+    First columns:Integer=1
```

Comment:
Maybe
```suggestion
    ## Add the first specified number of columns.
    First columns:Integer=1
```
?


I don't think `Columns_To_Add` is (at least currently) used in "read" contexts?